### DISCUSSION TITLE
Will there be a new iso image

### DISCUSSION THREAD

**Discussion by w18g** [regolith-linux/regolith-desktop#1028] [General]
based on ubuntu 24.04.1 LTS (eta 8/15)?

Need to plan either an upgrade from 22.04 or fresh install....argh

**Comment by kgilmer** [comment]
Hi @w18g , currently I have no plan to produce an ISO based on Ubuntu 24.04.  (Testing these is very time intensive.)  Recently we've been focusing on a closer integration with upstream Debian.  I expect at some point soon we'll have a Debian-based ISO installer, but that's probably not what you're looking for.  I suggest installing Ubuntu 24.04 and then the Regolith desktop inside of that.  If you want to make it more streamlined as the Regolith ISO you can install `regolith-lightdm-config` which will pull in the `lightdm` display manager.  Then you could remove `gdm3`, `gnome-shell`, etc. if you like.

**Comment by w18g** [comment]
ok thanks. Will actually be fine with a debian based iso.
